intake/container/persist.py

```diff
@@ -49,12 +49,12 @@ def __new__(cls, *args, **kwargs):
 
     def __init__(self, path=None):
         # from fsspec.registry import filesystem
-        from dask.bytes.core import get_fs
+        from dask.bytes.core import get_fs_token_paths
```

Comment:
I think `from fsspec import filesystem`, and `filesystem(protocol)` are most appropriate